logout/index.html

```diff
@@ -0,0 +1,334 @@
+          <img
+            class="img-responsive border-rad-md"
+            src="/logout/images/phone.jpg"
+            alt="mountain-lake-northern-lights"
+          />
+          <h3>Samsung Galaxy S21 FE</h3>
+          <p>⭐⭐⭐⭐</p>
+          <div class="price-container list-inline">
+            <p class="text-wt-semibold">
+              45,999<span class="text-size-xs text-style-strike margin-md"
+                >51,999</span
+              >
+            </p>
+            <p class="text-wt-bold">12% OFF</p>
+          </div>
+          <span class="card-badge card-badge-new border-rad-md">NEW</span>
+          <div class="card-btn-wrapper">
+            <span
+              ><button class="btn btn-primary text-size-sm">
+                ADD TO CART
+              </button></span
+            >
+            <span>
+              <button class="btn btn-outline text-size-sm">
+                <i class="fa-regular fa-heart"></i>ADD TO WISHLIST
+              </button>
+            </span>
+          </div>
+        </div>
+        <div class="card card-vertical box-shadow border-rad-md">
+          <img
+            class="img-responsive border-rad-md"
+            src="/logout/images/phone.jpg"
+            alt="mountain-lake-northern-lights"
+          />
+          <h3>Samsung Galaxy S21 FE</h3>
+          <p>⭐⭐⭐⭐</p>
+          <div class="price-container list-inline">
+            <p class="text-wt-semibold">
+              45,999<span class="text-size-xs text-style-strike margin-md"
+                >51,999</span
+              >
+            </p>
+            <p class="text-wt-bold">12% OFF</p>
+          </div>
+          <span class="card-badge card-badge-new border-rad-md">NEW</span>
+          <div class="card-btn-wrapper">
+            <span
+              ><button class="btn btn-primary text-size-sm">
+                ADD TO CART
+              </button></span
+            >
+            <span>
+              <button class="btn btn-outline text-size-sm">
+                <i class="fa-regular fa-heart"></i>ADD TO WISHLIST
+              </button>
+            </span>
+          </div>
+        </div>
+        <div class="card card-vertical box-shadow border-rad-md">
+          <img
+            class="img-responsive border-rad-md"
+            src="/logout/images/phone.jpg"
+            alt="mountain-lake-northern-lights"
+          />
+          <h3>Samsung Galaxy S21 FE</h3>
+          <p>⭐⭐⭐⭐</p>
+          <div class="price-container list-inline">
+            <p class="text-wt-semibold">
+              45,999<span class="text-size-xs text-style-strike margin-md"
+                >51,999</span
+              >
+            </p>
+            <p class="text-wt-bold">12% OFF</p>
+          </div>
+          <span class="card-badge card-badge-new border-rad-md">NEW</span>
+          <div class="card-btn-wrapper">
+            <span
+              ><button class="btn btn-primary text-size-sm">
```

Comment:
Instead of text-size-sm, you can write text-sm, its obvious you are using a smaller size there.

Reply:
Yes I'm kinda stuck with that name now because that's in the CL.